src/cairoUtilFuncGen/memory/implicitConversion.ts

```diff
@@ -441,14 +417,11 @@ export class MemoryImplicitConversionGen extends StringIndexedFuncGen {
     if (targetType.signed && targetType.nBits !== sourceType.nBits) {
       const conversionFunc = `warp_int${sourceType.nBits}_to_int${targetType.nBits}`;
       return [
-        `let (${targetVar}) = ${conversionFunc}(${sourceVar});`,
+        `let ${targetVar} = ${conversionFunc}(${sourceVar});`,
         [this.requireImport(INT_CONVERSIONS, conversionFunc)],
       ];
     } else if (!targetType.signed && targetType.nBits === 256 && sourceType.nBits < 256) {
-      return [
-        `let (${targetVar}) = felt_to_uint256(${sourceVar});`,
-        [this.requireImport(...FELT_TO_UINT256)],
-      ];
+      return [`let ${targetVar}: u256 = ${sourceVar}.into();`, []];
```

Comment:
It requires an import of `Into` trait and the implementation of into trait

Reply:
yes, I think it would be a good issue to fix. Update the whole `generateScalingCode` family of funcs to Cairo 1. What do you think?